build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -168,10 +173,10 @@ function updateReporters(config) {
     config.reporters.push('coverage-istanbul');
   }
 
-  if (isTravisPushBuild() && JSON_REPORT_TEST_TYPES.has(config.testType)) {
+  if (isTravisPushBuild() && isReportTestType(config.testType)) {
```

Comment:
Ahh, I don't know if reporting test results belongs here :/ This should decoupled from the runtime test gulp tasks.

A clean way to do that is to create a separate gulp task, and then run it in our Travis job scripts that are found in `build-system/pr-check/`

Reply:
Understood -- could you point me to an example gulp task I could base it off of to get an idea?

Reply:
`build-system/tasks/check-owners.js` is a pretty simple gulp task you could potentially pattern-match off of